### DISCUSSION TITLE
BYOD exams with third-party applications, without IT team

### DISCUSSION THREAD

**Discussion by Spruce8913** [SafeExamBrowser/seb-server#94] [Q&A] [answered]
Dear SEB-programmers

First off, thank you for your hard work! We have successfully used (local) SEB for BYOD exams several times now at our department.

We are already thinking of extending this to using third-party applications in BYOD exams, as we simply do not have enough pc's to do this on managed computers. In [the documentation ](https://safeexambrowser.org/windows/win_usermanual_en.html#ApplicationsPane) the VDI setup is explained, where SEB server is installed and maintained by a central IT team. Unfortunately, our team consists mainly of teachers (no IT-experts), so I don't really see a way where we manage a similar VDI setup within our department.

So, do you offer services for setting up SEB server (hosting?, managing?, support with questions?)? If not, do you have any ideas on how we could organise BYOD exams with external applications? Or could you refer us some organisation that offers this (I encountered online for example https://www.schoolyear.com/ : any experience with them?)?

Kind regards in advance, and thank you for any pointers!

**Comment by Spruce8913** [comment]
As I need to notify my colleagues of what I think would be the best option (setting up VDI with SEB server ourselves, paying schoolyear.com or an unknown third option?) in the near future, I'm so bold to ping you: @danschlet , @dbuechel , @anhefti .

If it's quicker for you, you can also contact me here: https://www.kuleuven.be/wieiswie/nl/person/00119565

**Reply by danschlet** [reply, marked as answer]
I know only this company offering an assessment solution providing cloud-based third party applications combined with SEB, which seems to be targetting a local Swiss German market: https://smartlearn.one/

We are evaluating if we should include direct support for VDI and cloud-based virtual machines into SEB Server, but as we are working on many other tasks, we can't estimate when we would start working on something like that. In case you want to emphazise or contribute to such a feature request, you can write to alliance@safeexambrowser.org.

**Reply by Spruce8913** [reply]
Thank you for your response! I will discuss the possibilities within my team.
